Sources/NIOSSL/NIOSSLClientHandler.swift

```diff
@@ -31,6 +31,11 @@ private extension String {
 /// This handler can be used in channels that are acting as the client
 /// in the TLS dialog. For server connections, use the `NIOSSLServerHandler`.
 public final class NIOSSLClientHandler: NIOSSLHandler {
+    public convenience init(context: NIOSSLContext, serverHostname: String?) throws {
+        try self.init(context: context, serverHostname: serverHostname, optionalCustomVerificationCallback: nil)
+    }
+
+    @available(*, deprecated, renamed: "init(context:serverHostname:customVerificationCallback:)")
```

Comment:
This isn't strictly a rename because the callback type changed, right?

Reply:
That's true, but it provides a useful fixit.